front/app/components/SignUpIn/AuthProviderButton.tsx

```diff
@@ -208,11 +229,12 @@ const AuthProviderButton = memo<Props>(
                   privacyPolicyError={privacyError}
                   onTacAcceptedChange={handleTacAcceptedChange}
                   onPrivacyAcceptedChange={handlePrivacyAcceptedChange}
+                  authProvider={authProvider}
                 />
                 <ButtonWrapper>
                   <ContinueButton
                     onClick={handleContinueClicked}
-                    disabled={!(tacAccepted && privacyAccepted)}
+                    disabled={!isContinueEnabled}
```

Comment:
Did you test the case when `isContinueEnabled` is false? Maybe it's worth doing it.